### PR TITLE
Does anyone read this? I'll be at the coffee shop accross the street.

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

**Comment by brianamarie** [comment]
@butlerx Coffee sounds good... ☕

File: butlerx.md [hunk 1]
```diff
@@ -0,0 +1 @@
+ will you cover templates or working on fork pull requests
```
Comment (iluvk9s) [inline butlerx.md]: Good question! @JamiesHQ - this may be of interest to you :-)

**Comment by brianamarie** [comment]
@butlerx We will cover the fork/pull workflow tomorrow 😄

**Comment by bbusche** [comment]
In case you get a fork stuck in your arm, you can request somebody to pull it out for you.

😆

**Pull request merged**